### ISSUE TITLE
JAWS dropdown announces selection item text twice 

### ISSUE THREAD

**Issue by sugandhi-ms** [OfficeDev/office-ui-fabric-react#9053]
## Codepen demonstrating the issue
Using this codepen: https://codepen.io/jdh-msft/pen/rgxqev?editors=0010

## Actual behavior:
When using office-fabric-react Dropdown, JAWS announces the selected text twice on tabbing to the dropdown and when the selection is changed using enter/space keys,
Browser: Chrome

## Expected behavior:
JAWS screen reader should announce the selected item once

**Comment by jdhuntington** [comment]
Thanks @sugandhi-ms - will look into this.

**Comment by jdhuntington** [comment]
@sugandhi-ms I am unable to repro. Tried with both JAWS and Narrator, it seems that the selection is only being read  a single time. Are you still seeing this issue?
